app/views/subnets/index.html.erb

```diff
@@ -24,7 +24,14 @@
         <td class="ellipsis"><%= subnet.domains.map(&:name).to_sentence %></td>
         <td><%= subnet.vlanid %></td>
         <td class="ellipsis"><%= subnet.dhcp %></td>
-        <td><%= link_to_if_authorized(hosts_count[subnet], hash_for_hosts_path(:search => "subnet.name=\"#{subnet}\"")) %>
+        <td>
+          <%=
+            # TODO: this only works for IPv4
+            count = hosts_count[subnet]
```

Comment:
I think this comes from:

https://github.com/theforeman/foreman/blob/59bf491958634f067e7c4da16b2050a44e72c2db/app/helpers/application_helper.rb#L327-L329

Which in turn comes from [HostCounter](https://github.com/theforeman/foreman/blob/develop/app/services/host_counter.rb).